server/src/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nelnatsol/pyroengine:latest
+FROM pyronear/armv7_pyro-engine:latest
```

Comment:
For posterity, we need to update the tag: on dockerhub, we should have 1 repo per github repo (pyroengine)
The tag should point to the version of it

Then you can add the CPU arch if you want in the tag, but there is no use. Docker automatically retrieves the image that fits your arch. Here is an example:
https://hub.docker.com/r/pytorch/pytorch/tags?page=1&ordering=last_updated
pytorch image have the "linux/amd64" arch
https://hub.docker.com/r/pyronear/armv7_pyro-engine/tags?page=1&ordering=last_updated
this one has "linux/arm"

Reply:
Okay, interesting !
Hence, if I have understood, just need to have the two archs (armv7 and linux) of pyro-engine image on pyroengine dockerhub ?

Reply:
@fe51 Yes but for two different use cases 

@frgfm Ok I get how docker hub works now. I was afraid to delete the other image